descope/management/user.py

```diff
@@ -1055,6 +1064,9 @@ def _compose_create_body(
             test,
             picture,
             custom_attributes,
+            None,
```

Comment:
Nicer to call the method with params names, so you dont need to add the extra none

Reply:
good point, fixed